manifest.dnn

```diff
@@ -66,14 +66,6 @@
             </script>
           </scripts>
         </component>
-        <component type="ResourceFile">
-          <resourceFiles>
-            <basePath>bin</basePath>
-            <resourceFile>
-              <name>symbols.zip</name>
-            </resourceFile>
-          </resourceFiles>
-        </component>

```

Comment:
Any reason to remove this ?

Reply:
Why would that be useful in production?

Reply:
It doesn’t necessarily harm things. Especially given the current state of this project

Reply:
It provides better error logging and does not affect performance...

Reply:
Thanks for the changes